src/main/native/windows/processes-jni.cc

```diff
@@ -172,6 +172,288 @@ class NativeProcess {
     }
   }
 
+  void Create(JNIEnv* env, jstring java_argv0, jstring java_argv_rest,
```

Comment:
change return type to `bool` and return false on errors?

Reply:
Done - but the JNI interface method (nativeCreateProcess) doesn't seem to have a mechanism to report errors back, so I'm not sure what to do with the result in that function. (Maybe we should rework this to use exceptions in a follow-up? I think some of our JNI code already does that so we could learn how throwing Java exceptions from JNI works from there :))

Reply:
Thanks, but I should've clarified that I meant a C++ `bool` type, not `jboolean`.

Other JNI methods that I wrote have a String array argument, which they expect to be a Java-allocated String[1], and which the C++ function writes an error message into (if there's any) and returns false or some error code. The Java side of course needs to check this array after the call.

I suppose we could also use `env->ThrowNew` although that requires loading a throwable class. I'm not sure if we can throw checked exceptions.

I'm fine with either approach. WDYT of them?

Reply:
Why not just let the error handling work the same as before? I might be missing something, but what the code here does is just exactly what the old code did - it's not super pretty, but it works well enough (the method sets the error_ member variable and the Java code checks via `processGetLastError` whether there was an error and if yes throws an exception).

I'm super open to making this code better and using better mechanisms for error reporting, but should we do it in this CL, which is already big enough?

Reply:
SGTM